src/com/article-changer-v2.js

```diff
@@ -80,7 +84,7 @@
 				var cleanUrl = loc.href.substring(loc.hostname.length + loc.protocol.length + 2);
 				
 				App.mediator.notify('pageLoad.end');
-				App.mediator.notify('articleChanger.entering', {url : cleanUrl, data : dataLoaded});
+				App.mediator.notify('articleChanger.loaded', {url : cleanUrl, data : dataLoaded});
```

Comment:
Can you format this piece ? on multiple lines... without the space before the `:`